helix-view/src/editor.rs

```diff
@@ -104,6 +104,46 @@ impl Default for FilePickerConfig {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case", default, deny_unknown_fields)]
+pub struct ExplorerConfig {
+    /// explorer style
+    /// 0b00 embed + tree
+    /// 0b01 embed + list
+    /// 0b10 overlay + tree
+    /// 0b11 overlay + list
+    pub style: usize,
```

Comment:
Is it a good idea to do usize rather than using list of string?

Reply:
I would use two enums here:

```rust
enum ExplorerStyle {
  Tree,
  List
}

enum ExplorerPosition {
  Embed,
  Overlay
}
```

We could also use `bitflags` since it's already a dependency, but I think it's not worth it